guides/common/modules/con_updating-project-to-next-minor-version.adoc

```diff
@@ -0,0 +1,10 @@
+[id="updating-project-to-next-minor-version_{context}"]
+= Updating {Project} to the Next Minor Version
+
+Use this guide to update your {ProjectServer} and {SmartProxyServer} to a new minor release version, such as from {ProjectVersion}.0 to {ProjectVersion}.1.
+
+The minor releases patch security vulnerabilities and minor issues discovered after code is released, and are often fast and non-disruptive to your operating environment.
+
+.Prerequisite
+* Back up your {ProjectServer} and all {SmartProxyServers}.
+For more information, see {AdministeringDocURL}backing-up-{project-context}-server-and-{smart-proxy-context}_admin[Backing Up {ProjectServer} and {SmartProxyServer}] in _{AdministeringDocTitle}_.
```

Comment:
I suggest distributing the prerequisite into all sections instead of having it here.

Reply:
Added to all the three sessions +1

Reply:
Added to all the three sections :+1: